packages/fxa-settings/src/components/LinkRememberPassword/index.tsx

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { Localized } from '@fluent/react';
+import { Link } from '@reach/router';
+
+type LinkRememberPasswordProps = { email?: string; forceEmail?: string };
+
+const LinkRememberPassword = ({
+  email,
+  forceEmail,
+}: LinkRememberPasswordProps) => {
+  const linkTarget = email ? '/signin' : '/force_auth';
+  const queryParam = email || forceEmail || '';
+
+  return (
+    <div className="text-sm mt-6">
+      <Localized id="remember-pw-link">
+        <Link
+          to={linkTarget + `?email=` + queryParam}
```

Comment:
Q: do we need to escape the email address if passing it along a query string?
For example, would "foo+bar@gmail.com" pass correctly?